index.html

```diff
@@ -176,6 +176,21 @@ <h3 class="lang-font">Skills</h3>
           </div>
         </div>
     </section>
+    <section class="contact-section">
+      <div class="contact-form">
+      <h2 class="con-title">Contact me</h2>
+      <p class="con-paragraph">If you have an application you are interested in developing, a feature that
+          you
+          need built or a project that needs coding. I’d love to help with it</p>
+      <form action="https://formspree.io/f/mrgjqpvb" method="POST" class="con-form">
+          <input type="text" name="_name" value="Yeremias" maxlength="30" class="form-text" placeholder="name" required>
+          <input type="email" name="_reply" value="natajayanj14@gmail.com" class="form-text" placeholder="email" required>
+          <textarea name="_message" placeholder="Write your message here" maxlength="500"
+              class="form-text message" required></textarea>
+          <button class="con-but" type="submit">Get in touch</button>
+      </form>
+    </div>
```

Comment:
## OPTIONAL 

![abc](https://user-images.githubusercontent.com/69638013/154313446-39e10c68-4522-4e8c-9296-a0759946f73e.PNG)

- [ ] Kindly add the background icons check screen shot above

Reply:
Hey @prolajumokeoni thank you so much. will work on the icon background .

thank you once more